app/views/course/assessment/assessments/show.html.slim

```diff
@@ -12,12 +12,12 @@
 
   table.table.table-bordered.details-table
     tbody
-      tr.maximum_experience_points
-        th = t('.maximum_experience_points')
-        td = @assessment.base_exp + @assessment.time_bonus_exp + @assessment.extra_bonus_exp
-      /tr.maximum_grade
-        th = ('.maximum_grade')
-        td = @assessment.maximum_grade
+      tr.base_exp
+        th = t('.base_exp')
+        td = @assessment.base_exp
+      tr.bonus_exp
+        th = t('.bonus_exp')
+        td = @assessment.time_bonus_exp + @assessment.extra_bonus_exp
```

Comment:
The second type is actually not used yet. The behaviour is not defined yet, we might want to discuss that.